packages/icomoon/react-native-vector-icons-icomoon.podspec

```diff
@@ -0,0 +1,79 @@
+require 'json'
+
+package = JSON.parse(File.read(File.join(__dir__, 'package.json')))
+
+Pod::Spec.new do |s|
+  s.name         = 'react-native-vector-icons-icomoon'
+  s.version      = package['version']
+  s.summary      = package['description']
+  s.homepage     = package['homepage']
+  s.license      = package['license']
+  s.authors      = package['author']
+
+  s.platforms    = { ios: min_ios_version_supported, tvos: '9.0', visionos: '1.0' }
+  s.source       = { git: package['repository']['url'], tag: "v#{s.version}" }
+
+  s.resources = 'fonts/*.ttf'
+
+  s.script_phase = {
+    name: 'Copy Custom Fonts',
+    execution_position: :after_compile,
+    script: <<~SCRIPT
```

Comment:
this seems wrong? This shouldn't be here and instead there should be`s.resources = 'fonts/*.ttf'` only?

Reply:
See my other comment on the template that generates this